test/test_known_epitopes.py

```diff
@@ -43,5 +44,5 @@ def test_HIV_epitope():
     for mhc_class in mhc_classes:
         mhc_model = mhc_class("HLA-A*02:01", epitope_lengths=9)
         epitope = mhc_model.predict("SLYNTVATL")[0]
-        assert epitope.value < 500, \
```

Comment:
Did you figure out what happened? Where did the change to percentile rank come from?